src/core/ipv4/ip4_napt.c

```diff
@@ -137,8 +159,11 @@ napt_debug_print(void)
 static void
 ip_napt_deinit(void)
 {
-  mem_free(ip_napt_table);
-  mem_free(ip_portmap_table);
+  napt_list = NO_IDX;
+  napt_stats.max_entries = 0;
+  ip_portmap_max = 0;
+  if (ip_napt_table) mem_free(ip_napt_table);
```

Comment:
we assume `free(NULL)` is a valid statement, the same for the line below.
```suggestion
  mem_free(ip_napt_table);
```